Surface_mesh_skeletonization/include/CGAL/Mean_curvature_flow_skeletonization.h

```diff
@@ -513,6 +513,28 @@ std::size_t collapse_short_edges();
     m_omega_P = value;
   }
 
+  /// \cgalAdvancedFunction
+  /// \cgalAdvancedBegin
+  /// Fixes a range of vertices.  Fixed vertices will not be moved
```

Comment:
```suggestion
  /// sets the vertices in the range `[begin, end[` as fixed.  Fixed vertices will not be moved
```

Reply:
Is the notation with two opening square brackets intentional, or is it maybe a typo?

Reply:
French notation sorry

Reply:
Ah, I just noticed that all closed ranges in the documentation are indeed written like that.  But come to think of it, technically, it should be `[begin, end)`, since that's the way it's interpreted by the set's constructor, shouldn't it?

Reply:
end is a past-the-end.

Reply:
I have incorporated the change as `[begin, end)`